### PR TITLE
Track the pushed branch

### PR DESCRIPTION
**Pull request by rgarrigue**
Solve #112 : automatically track the pushed branch, so after "git pull-request", the PR can be worked on with "git pull" & "git push"

### CODE REVIEW

File: git_pull_request/__init__.py [hunk 1]
```diff
@@ -622,7 +622,7 @@ def fork_and_push_pull_request(
             [
                 "git",
                 "push",
-                "-f",
+                "-uf",
```
Comment (jd) [inline git_pull_request/__init__.py:625]: Can we use the long form of the option? That makes it easier to understand when reading.

Comment (rgarrigue) [reply git_pull_request/__init__.py]: Ok, fixed

**Comment by rgarrigue** [comment]
@jd do you mind releasing to pypi soon ?

**Comment by jd** [comment]
Done.

**Pull request merged**